client/state/users/suggestions/schema.js

```diff
@@ -0,0 +1,24 @@
+export const itemsSchema = {
+	type: 'object',
+	additionalProperties: false,
+	patternProperties: {
+		'^\\d+$': {
+			type: 'array',
+			items: {
+				type: 'object',
+				additionalProperties: false,
```

Comment:
We can relax this a bit, and remove the `additionalProperties` here. If we add a property to the users suggestions object that that client doesn't use, we'd be throwing out data that could still be used.